src/plugins/lua/luaplugin.cpp

```diff
@@ -44,15 +44,41 @@
 //#define POLYGON_FORMAT_PAIRS
 //#define POLYGON_FORMAT_OPTIMAL
 
+namespace {
+
+    void writeColor(Lua::LuaTableWriter &writer,
+                    const char *name,
+                    const QColor &color)
+    {
+        // Example: backgroundcolor = { 255, 200, 100 }
+        writer.writeStartTable(name);
+        writer.setSuppressNewlines(true);
+        writer.writeValue(color.red());
+        writer.writeValue(color.green());
+        writer.writeValue(color.blue());
+        if (color.alpha() != 255)
+            writer.writeValue(color.alpha());
+        writer.writeEndTable();
+        writer.setSuppressNewlines(false);
+    }
+}
+
 using namespace Tiled;
 
 namespace Lua {
 
-LuaPlugin::LuaPlugin()
+void LuaPlugin::initialize()
+{
+    addObject(new LuaMapFormat(this));
+    addObject(new LuaTilesetFormat(this));
+}
+
+bool LuaMapFormat::supportsFile(const QString &fileName) const
 {
+    return true;
```

Comment:
@bjorn, can you please provide some direction on how implement this. I could copy as best I can from other places but it'd be good to not waste my time doing the wrong thing if possible ;)